content/en/_index.html

```diff
@@ -14,6 +14,10 @@
       Contribute
       <i class="fas fa-pencil-alt ml-1"></i>
     </a>
+    <a class="btn btn-lg btn-info mx-1 mb-3 mb-sm-0 px-3 px-md-4 py-2" href="/docs/genai/">
+      GenAI 
+      <i class="fas fa-brain ml-1"></i>
+    </a>
```

Comment:
It's important to not have too many options for the "intended action" when a user comes to a homepage.

I suggest if we want people to know about GenAI, we have a section under the Introduction/Getting Started which is already linked in its own button, and leave the two buttons as "Get Started" and "Contribute".

Reply:
I disagree here. I think Kubeflow has lost mind-share in the GenAI era. We received explicit feedback from the outgoing KSC that we need to make Kubeflow more visible in the GenAI conversation. This PR was my first attempt to do that.

Reply:
I disagree it's bad UX.

Buttons are a mechanism to (1) succinctly give users information and (2) redirect them to more information. 

Usually people prioritize real estate above the fold on whatever is most important to the top of funnel and my guess is that based on the current state of AI, GenAI (and Kubeflow's ability to support those needs) is a good fit for it.

CC @varodrig @chasecadet @StefanoFioravanzo  thoughts?